Makefile

```diff
@@ -1,67 +1,65 @@
-TEST?=$$(go list ./... | grep -v 'vendor' | grep -v ci-tests)
 GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
 WEBSITE_REPO=github.com/hashicorp/terraform-website
 PKG_NAME=kubevirt
-
-GOOS=$(shell go env GOOS)
-GOARCH=$(shell go env GOARCH)
 ifeq ($(OS),Windows_NT)  # is Windows_NT on XP, 2000, 7, Vista, 10...
-	DESTINATION=$(APPDATA)/terraform.d/plugins/$(GOOS)_$(GOARCH)
+	DESTINATION_PREFIX=$(APPDATA)/terraform.d/plugins
 else
-	DESTINATION=$(HOME)/.terraform.d/plugins/$(GOOS)_$(GOARCH)
+	DESTINATION_PREFIX=$(HOME)/.terraform.d/plugins
 endif
 
+export BIN_DIR=$(CURDIR)/build/_output/bin
+export GOROOT=$(BIN_DIR)/go
+export GOBIN=$(GOROOT)/bin
+export GO=$(GOBIN)/go
+export GOFMT=$(GOBIN)/gofmt
+
 all: test install
 
-clean:
-	go clean
-	@echo "==> Removing $(DESTINATION) directory"
-	@rm -rf $(DESTINATION)
+$(GO):
```

Comment:
This directive will be called multiple times during the run of the CI.
You can create a marker file using the "install-go.sh" script and in the directive check for the existence of the file, if it exist don't run the command.

Reply:
The $(GO) target in the Makefile is executed only if the file doesn't exist
Makefile already solve this issue for us